FirebasePerformance/Sources/Instrumentation/FPRNetworkTrace.m

```diff
@@ -91,7 +91,7 @@ - (nullable instancetype)initWithURLRequest:(NSURLRequest *)URLRequest {
 
   NSString *trimmedURLString = [FPRNetworkTrace stringByTrimmingURLString:URLRequest];
   if (!trimmedURLString || trimmedURLString.length <= 0) {
-    FPRLogInfo(kFPRNetworkTraceURLLengthExceeds, @"URL length outside limits, returning nil.");
+    FPRLogWarning(kFPRNetworkTraceURLLengthExceeds, @"URL length outside limits, returning nil.");
```

Comment:
Same here. I don't think this is a warning for the developer. This is more informational for the developer and would be helpful only for our debugging and not for the developer.

Reply:
If I'm understanding correctly when this gets logged, I completely disagree. We are warning the developer that the url is too long and it won't get recorded.

Reply:
Discussed offline. Will log this message with `Warning` because a product feature may stop functioning correctly